frontend/ui/data/creoptions.lua

```diff
@@ -1,7 +1,7 @@
-local Device = require("device")
 local S = require("ui/data/strings")
 local optionsutil = require("ui/data/optionsutil")
 local _ = require("gettext")
+local Device = require("device")
```

Comment:
The coding style isn't set in stone of course, but if we go against it I figure there should be a rationale to chang or amend it. ;-)

Reply:
yeah, sorry that I forgot we have this style in place :P